app/assets/javascripts/discourse/templates/components/ip-lookup.hbs

```diff
@@ -6,7 +6,11 @@
 {{#if show}}
   <div class="location-box">
     <a class="close pull-right" {{action "hide"}}>{{d-icon "times"}}</a>
-    <a class="btn pull-right" {{action "copy"}}>{{d-icon "copy"}}</a>
+    {{#if copied}}
+      <a class="btn btn-hover pull-right" {{action "copy"}}>{{d-icon "copy"}} {{i18n "ip_lookup.copied"}}</a>
```

Comment:
I don't think we need the `action` here.

Reply:
`action` is not necessary indeed, but maybe some people are successful in messing their clipboard in less than 2 seconds and are impatient to copy again.

I have no problems in removing it though. :)